packages/eslint-plugin/rules/acronym-case.js

```diff
@@ -78,6 +78,12 @@ module.exports = {
 					return;
 				}
 
+				// Ignore identifiers that are a function call or argument, we can assume the identifier will be validated at the point of declaration,
+				// but want to allow those which are exceptions to the rule to be invoked or passed into functions without raising a linting error.
+				if ( node.parent?.type === 'CallExpression' ) {
+					return;
+				}
```

Comment:
With this code being removed
https://github.com/google/site-kit-wp/blob/c92f822d765eb09952b1b497b6b11220f028d847/assets/js/components/notifications/GA4AdSenseLinkedNotification.js#L220-L223

we currently seem to have 100 `// eslint-disable-line sitekit/acronym-case` suppression across 41 files in our codebase. Almost all of them seems to have related to properties like `accountId`, `containerId`, `publicationId` etc.

I think we can avoid adding this rule as of now because it’s quite broad and can skip more cases than intended. It essentially ignores any identifier inside a function call, which might suppress errors we actually want to catch in different contexts.

Reply:
Thanks @ankitrox, I did consider this too while executing the issue. I came to the conclusion it's still useful to implement, and I discussed this under [**relevant technical choices**](https://github.com/google/site-kit-wp/pull/9618). Please take a look, this also links to the comment where the design choice for this rule was originally discussed.